presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -35,7 +35,8 @@
 
     public TestingPostgreSqlServer()
     {
-        dockerContainer = new PostgreSQLContainer("postgres:12.4")
+        // Use the oldest supported PostgreSQL version
+        dockerContainer = new PostgreSQLContainer("postgres:9.5")
```

Comment:
We should extract this as constant somewhere. Should we document it as well?

Reply:
see above